### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by grzegorz-pietrzak-droptica**

### CODE REVIEW

**Review by rafal-sereda-droptica — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,55 +1,37 @@
...
 * **Demo**: [droopler-demo.droptica.com](https://droopler-demo.droptica.com)
 * **Composer template**: [github.com/droptica/droopler_project](https://github.com/droptica/droopler_project)
 * **Drupal.org project**: [drupal.org/project/droopler](https://www.drupal.org/project/droopler)
 
 For the latest news please subscribe to our [Facebook](https://www.facebook.com/Droopler/) and [Twitter](https://twitter.com/DrooplerCMS).
 
-## What's in this repository? ##
-This repository contains a Drupal profile. When you put it in your /profiles directory, the Drupal installer gets modified and installs base Droopler theme, some module dependencies, and demo content.
+## What's in this repository?
+This repository contains a Drupal profile. When you put it in the `/profiles/contrib/droopler` directory, the Drupal installer gets modified and installs base Droopler theme, some module dependencies, and demo content.
```
Comment (mariusz-andrzejewski-droptica) [inline README.md:16]: contains Drupal czy Droopler profile?

Comment (rafal-sereda-droptica) [reply README.md:16]: but Droopler **is** a Drupal profile :D

Comment (rafal-sereda-droptica) [reply README.md:16]: > and installs base Droopler theme, some module dependencies, and demo content.

@grzegorz-pietrzak-droptica  but maybe we should mention here that there is an option to choose some "optional module and/or demo content" ?

Comment (mariusz-andrzejewski-droptica) [reply README.md:16]: No tak, problem kwadratu i prostokąta  :D
Każdy Droopler jest Drupalem ale nie każdy Drupal drooplerem.
W kazdym razie tylko chcialem sie co do tego upenwić

**Review by mariusz-andrzejewski-droptica — APPROVED** [review]

**Pull request merged**